python/core/auto_generated/vectortile/qgsmapboxglstyleconverter.sip.in

```diff
@@ -347,7 +347,7 @@ Parses a numeric value which is interpolated by zoom range.
          - defaultNumber: storage for a reasonable "default" number representing the overall property.
 %End
 
-    static QgsProperty parseInterpolateOpacityByZoom( const QVariantMap &json, int maxOpacity );
+    static QgsProperty parseInterpolateOpacityByZoom( const QVariantMap &json, int maxOpacity, QgsMapBoxGlStyleConversionContext &context );
```

Comment:
Adding the new argument here is an api break -- can you make it optional for these methods?

Reply:
Done. Also remove the comment about private API here?